site/multiembedder/MultiEmbedder.tsx

```diff
@@ -247,11 +283,20 @@ class MultiEmbedder {
 
         const configUrl = `${GRAPHER_DYNAMIC_CONFIG_URL}/by-uuid/${view.fullConfigId}.config.json`
 
-        await this._renderGrapherComponentIntoFigure(figure, {
-            configUrl,
-            embedUrl,
-        })
+        if (embedUrl.queryParams.hideControls === "true") {
+            await this._renderGrapherComponentIntoFigure(figure, {
```

Comment:
The above assignments to `view` and `configUrl` we can pull into this if statement block now.

Reply:
Yep! Moved.